include/dt-bindings/reset/aspeed,ast1700-reset.h

```diff
@@ -0,0 +1,69 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+/*
+ * Device Tree binding constants for AST2700 reset controller.
+ *
+ * Copyright (c) 2023 Aspeed Technology Inc.
+ */
+
+#ifndef _MACH_ASPEED_AST1700_RESET_H_
```

Comment:
Same as above, AST1700

Reply:
We need these changes to enable LTPI i2c/I3c